### PR TITLE
Csp 911 admin manage events search page

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public async Task<IActionResult> Get(Guid calendarEventId, [FromHeader(Name = Co
 
     [HttpGet]
     [ProducesResponseType(typeof(GetCalendarEventsQueryResult), StatusCodes.Status200OK)]
-    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs:44]: We should return empty results when no data matches the filters.

**Review by DevenKShah — CHANGES_REQUESTED** [review]
NotFound is incorrect response for where a list is expected. 

Also a little bit changes on the approach of requested member header.

File: src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs:44]: ok

File: src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs [hunk 2]
```diff
@@ -41,7 +41,7 @@ public async Task<IActionResult> Get(Guid calendarEventId, [FromHeader(Name = Co
...
     public async Task<IActionResult> GetCalendarEvents([FromQuery] GetCalendarEventsModel model, CancellationToken cancellationToken)
```
[the remark this answers is not in the document]
Comment (MarkFCain) [reply src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs:45]: It stops working without the attribute, I think it's needed to clearly map the model to the querystring inputs, so leaving as is.

File: src/SFA.DAS.AANHub.Application/CalendarEvents/Queries/GetCalendarEvents/GetCalendarEventsQueryValidator.cs
[the remark this answers is not in the document]
Comment (MarkFCain) [reply src/SFA.DAS.AANHub.Application/CalendarEvents/Queries/GetCalendarEvents/GetCalendarEventsQueryValidator.cs:1]: PutAttendanceCommand also uses it

[the remark this answers is not in the document]
Comment (MarkFCain) [reply src/SFA.DAS.AANHub.Application/CalendarEvents/Queries/GetCalendarEvents/GetCalendarEventsQueryValidator.cs:1]: also GetMemberAttendances, GetCalendarEventById

**Review by DevenKShah — APPROVED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**